runtimevar/etcdvar/etcdvar.go

```diff
@@ -38,11 +38,12 @@ import (
 // It is provided for future extensibility.
 type Options struct{}
 
-// New constructs a runtimevar.Variable object that uses client to watch
-// variables in etcd.
-// Provide a decoder to unmarshal updated configurations into similar
-// objects during the Watch call.
-func New(name string, cli *clientv3.Client, decoder *runtimevar.Decoder, _ *Options) (*runtimevar.Variable, error) {
+// New constructs a runtimevar.Variable that uses client to watch the variable
+// name on an etcd server.
+// Etcd returns raw bytes; provide a decoder to decode the raw bytes into the
```

Comment:
I think etcd is always lowercased.

```suggestion
// etcd returns raw bytes; provide a decoder to decode the raw bytes into the
```